xml/vt_docker_quick.xml

```diff
@@ -823,9 +823,51 @@ RUN zypper -n in vim</screen>
   </para>
 
   <para>
+    Portus is accessible for &slea;&nbsp;12 customers through the Containers
+    module. This means that you should perform the following command to install
+    Portus:
+  </para>
+
+  <screen>sudo zypper in Portus</screen>
+
+  <para>
+    After doing this, you should proceed on configuring your freshly installed
```

Comment:
Here it is not clear to me if this belongs to the step before (which also deals with configuration)? Maybe you can specify in the step above which sections of the external references there user is supposed to read and execute ( probably 'Portus’ dependencies' and 'Initial setup')?